emsdk

```diff
@@ -1644,6 +1649,9 @@ def main():
                                   passed to the emsdk activate command to
                                   activate the desired version.
 
+                   --enable-wasm: Try to enable WebAssembly support in the
```

Comment:
The wording was because at the time it only worked with `sdk-incoming`, but you are right, something more direct is better.